.github/workflows/generate-binaries.yml

```
@@ -19,24 +19,25 @@
-      branch-name: ${{ steps.extract_branch.outputs.branch }}
+
   generate-binaries:
     needs: generate-branch-name
     runs-on: ubuntu-latest
     env:
-      BRANCH_NAME: ${{ needs.generate-branch-name.outputs.branch-name }}
       GH_CREDENTIALS_PSW: ${{ secrets.GH_CREDENTIALS_PSW }}
       GIT_URL: echo ${env.GIT_URL}
     container:
-      image: gounthar/openscad:${{ needs.generate-branch-name.outputs.branch-name }}
+      image: gounthar/openscad:${{ env.BRANCH_NAME }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)